tests/lib.rs

```diff
@@ -639,3 +641,12 @@ fn protocol_stack() {
         assert_eq!(ps, toks);
     }
 }
+
+// Assert all `Protocol` variants are covered
+// in its `Arbitrary` impl.
+#[cfg(nightly)]
```

Comment:
Nicely done! At first I thought that we'd need the rust-version proc macro here but `cfg`s also work very well!